src/clang.rs

```diff
@@ -192,7 +197,8 @@ impl Cursor {
     /// Is the referent a fully specialized template specialization without any
     /// remaining free template arguments?
     pub fn is_fully_specialized_template(&self) -> bool {
-        self.is_template() && self.num_template_args() > 0
+        self.is_template() && self.num_template_args()
+                              .expect("Not a class template specialization") > 0
```

Comment:
can you call `unwrap` directly? It's clear from the backtrace what the failure is.